Dockerfile

```diff
@@ -11,14 +12,15 @@ FROM golang:1.16-alpine AS builder
 # Set necessary environment variables needed for our image.
 ENV CGO_ENABLED=1 GOOS=linux GOARCH=amd64
 
-RUN apk add libpcap-dev gcc g++ make
+RUN apk add libpcap-dev gcc g++ make bash
```

Comment:
@nimrod-up9  because [`build_extensions.sh`](https://github.com/up9inc/mizu/blob/b2abd7b990eb6bd6b14372b2f14f34e28248d572/build_extensions.sh) is executed in [here](https://github.com/up9inc/mizu/blob/b2abd7b990eb6bd6b14372b2f14f34e28248d572/Dockerfile#L44) and that's a Bash script (well not necessarily Bash syntax but see below), `bash` is needed.

[`build_extensions.sh`](https://github.com/up9inc/mizu/blob/b2abd7b990eb6bd6b14372b2f14f34e28248d572/build_extensions.sh) has `#!/bin/bash` shebang since the similar build scripts have that same shebang:
- [`build-agent-ci.sh`](https://github.com/up9inc/mizu/blob/b2abd7b990eb6bd6b14372b2f14f34e28248d572/build-agent-ci.sh#L1)
- [`build-push-featurebranch.sh`](https://github.com/up9inc/mizu/blob/b2abd7b990eb6bd6b14372b2f14f34e28248d572/build-push-featurebranch.sh#L1)

If there is `#!/bin/bash` shebang then there should be such an executable available in that path ideally IMO.

Also since the Dockerfile designed as "builder" it does not increase the image size. In fact `bash` [is already installed in the resulting image](https://github.com/up9inc/mizu/blob/a310953f05514d60c9ffb331f3c8554c8a2699af/Dockerfile#L44) in the default branch.

Reply:
Great answer. I missed the fact that bash is needed for `buld_extensions.sh`.